docs/reference/master-configuration-reference.md

```diff
@@ -128,5 +128,64 @@ Default: 1 hour.
 Example:
 
 ```yaml
-resyncPeriod=2h
+resyncPeriod: 2h
+```
+
+### leaderElection
+
+The `leaderElection` option exposes configuration to tweak leader election.
+
+#### leaseDuration
```

Comment:
Hmm, I'd suggest to rename this.
```suggestion
#### leaderElection.leaseDuration
```

This would follow the style used in [worker config reference](https://kubernetes-sigs.github.io/node-feature-discovery/v0.13/reference/worker-configuration-reference.html)

...and ditto for the other new options

Reply:
Changed.